Acknowledgements.tex

```diff
@@ -8,6 +8,96 @@ \chapter{Acknowledgements}
 Without the hard work and dedication of these people this document would not have been possible.
 The sections below list some of the active participants and organizations in the various PMIx standard iterations.
 
+%%%%%%%%%% Version 5.0
+\section{Version 5.0}
+
+The following list includes some of the active participants in the PMIx v5 standardization process.
+
+%TODO general decision, where is the cut-off point for alumnis involved prior to dec. 2020 (when we published v4.0), in particular people that participated in establishing ASC rules used for the first time in this document.
+
+\paragraph*{Release Managers for v5.0}
+
+\begin{itemize}
+    \item Ken Raffenetti
+    \item David Solt
+\end{itemize}
+
+\paragraph*{ASC Chairs terms during v5.0 preparation}
+
+\begin{itemize}
+    \item Aurelien Bouteiller (2023-2025)
+    \item Joshua Hursey (2019-2024)
+    \item Kathryn Mohror (2020-2023)
+    \item Ralph Castain (2018-2020)
+\end{itemize}
+
+\paragraph*{Working Group Chairs during v5.0 preparation}
+
+%TODO decide what to do with inactive WGs chairs, alumnis
+\begin{itemize}
+    \item David Solt (Implementation Agnostic Document Working Group)
+    \item Isaías A. Comprés (Tools Working Group)
+\end{itemize}
+
+\paragraph*{ASC Secretaries terms during v5.0 preparation}
+
+\begin{itemize}
+    \item Norbert Eicker (2023-2025)
+    \item Thomas Naughton (2020-2024)
+    \item Aurelien Bouteiller (2021-2023)
+    \item Stephen Herbein (2019-2021)
+\end{itemize}
+
+\paragraph*{Contributors}
+
+%TODO review list (copied from v4, updated with current reps and alts, all commit-authors, PR reviewers, etc should also appear here)
+\begin{itemize}
+    \item Ralph H. Castain
+    \item Joshua Hursey and David Solt
+    \item Dirk Schubert
+    \item John DelSignore
+    \item Aurelien Bouteiller
+    \item Michael A Raymond
+    \item Howard Pritchard
+    \item Brice Goglin
+    \item Kathryn Mohror
+    \item Thomas Naughton and Swaroop Pophale
+    \item William E. Allcock and Paul Rich
+    \item Michael Karo
+    \item Artem Polyakov
+    \item Ken Raffenetti and Shane Snyder
+    \item Norbert Eicker
+    \item Brice Goglin and Guillaume Mercier
+    \item Hari Subramoni and Nat Shineman
+    \item Martin Schulz
```

Comment:
I'm wondering if we should just alphabetize this list. Maybe we can add some notation to indicate which org or orgs and individual was associated with during the standardization timeframe.

Reply:
New commit authors are Brian Barrett and Stephen Herbein. To the best of my knowledge there were no new PR reviewers. I used the `gh` tool to list all PRs merged into master since 2020-09-30, which is when v4 was branched off.
```
raffenet@glitch:pmix-standard/ (authors@v5.0*) $ gh pr list --search "is:pr is:closed merged:>2020-09-30 base:master" --limit 2000 --json reviews > v5-reviews.json
raffenet@glitch:pmix-standard/ (authors@v5.0*) $ cat v5-reviews.json| jq | grep login | sort | uniq
          "login": "SteVwonder"
          "login": "abouteiller"
          "login": "dsolt"
          "login": "jjhursey"
          "login": "kathrynmohror"
          "login": "naughtont3"
          "login": "raffenet"
          "login": "rhc54"
          "login": "schulzm"
          "login": "shanedsnyder"
```